examples/with-neo4j/pages/movie/[title].js

```diff
@@ -33,13 +33,23 @@ export default function Movie() {
             </div>
             <div>
               <strong>Released: </strong>
-              {data.movie.released.low}
+              {data.movie.released.toNumber()}
             </div>
           </div>
           <div className="actors">
             <h2>Actors</h2>
             {data.movie.actors.map((actor) => (
-              <div key={actor}>{actor}</div>
+              <div key={actor}>
+                <Link
+                  key={actor}
+                  href="/actor/[name]"
+                  as={{
```

Comment:
You don't need to use `as` anymore - can just use this for the href! 😄

Reply:
OK great, consider it done!